.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+jobs:
+
+    test:
+        runs-on: ubuntu-latest
+        container:
+            image: python:3.10
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install dependencies.
+              run : |
+                python -m pip install typeguard -r requirements/requirements.python.test.txt
+
+            - name: Run tests.
+              run : |
+                python -m pytest tests -s --log-cli-level=info
+
+    install-as-package-and-test:
+        runs-on: [ubuntu-latest]
+        container:
+            image: python:${{ matrix.version }}
+        strategy:
+            matrix:
+                version: ['3.10', '3.12']
+        steps:
+            - name: Install as package.
+              run : |
+                  pip install git+https://github.com/uliegecsm/system-helpers.git@${{ github.sha }}
+
+            - name: Test 'apt' helpers as CLI directly.
+              run : |
+                  apt-helpers install-packages --update --clean --upgrade --packages jq
```

Comment:
Should this be followed by a test of some call of `jq`. Could be just asking to print out its help. The idea would be to make sure that it's installed.

Reply:
:heavy_check_mark: